### DISCUSSION TITLE
How to leverage the stabilizationWindowSeconds on autoscaling/v2beta2 for HPA

### DISCUSSION THREAD

**Discussion by adrianmiron** [kedacore/keda#1595] [Q&A / Need Help] [answered]
Hi,

I am a bit confused on how to get the KEDA (v1) controller to configure the HPA's created by it to use the  stabilizationWindowSeconds feature under the new behavior field for k8s 1.18   (https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#support-for-configurable-scaling-behavior) .

Looking at the created HPA's ( 1.5.0 chart version ) i see it leverage only     apiVersion: autoscaling/v2beta1 ; but even it did use autoscaling/v2beta2, how do i tell keda to pass it the field ? 

My scope is to lower the scaledown delay  ( x --> 1 ) from 5 minutes to 30 seconds.

Thanks

**Comment by tomkerkhove** [comment, marked as answer]
Adding @zroubalik but don't think we support this < v2.0

**Reply by adrianmiron** [reply]
Hi,

@tomkerkhove 
i moved to v2 and digged a bit through the crd's to get this done there. Thanks !

**Reply by zroubalik** [reply]
Yeah, you should definitely use v2 :)
